packages/utils/src/configured_bignumber.ts

```diff
@@ -1,9 +1,14 @@
 import { BigNumber } from 'bignumber.js';
 
-// By default BigNumber's `toString` method converts to exponential notation if the value has
-// more then 20 digits. We want to avoid this behavior, so we set EXPONENTIAL_AT to a high number
 BigNumber.config({
+    // By default BigNumber's `toString` method converts to exponential notation if the value has
+    // more then 20 digits. We want to avoid this behavior, so we set EXPONENTIAL_AT to a high number
     EXPONENTIAL_AT: 1000,
+    // Note(albrow): This is the lowest value for which
+    // `x.div(y).floor() === x.divToInt(y)`
+    // for all values of x and y <= MAX_UINT256, where MAX_UINT256 is the
+    // maximum number represented by the uint256 type in Solidity (2^256-1).
+    DECIMAL_PLACES: 78,
```

Comment:
We should never use `div(..)` or `floor()`. Always use `divToInt(..)`. But that is out of scope for this PR.

Reply:
AFAIK, we don't. This is for the benefit of our users just in case they do it.